File: packages/jexlang-java/src/main/java/io/github/vinaykumar0339/context/EvaluatorContext.java
package io.github.vinaykumar0339.context;

import io.github.vinaykumar0339.evaluator.JexEvaluator;

public class EvaluatorContext {
    private final JexEvaluator jexEvaluator;

    public  EvaluatorContext(JexEvaluator jexEvaluator) {
        this.jexEvaluator = jexEvaluator;
    }

    public JexEvaluator getJexEvaluator() {
        return jexEvaluator;
    }
}

File: packages/jexlang-swift/jexlang-swift/context/EvaluatorContext.swift
//
//  EvaluatorContext.swift
//  jexlang-swift
//
//  Created by Vinay Kumar on 06/12/25.
//

import Foundation

public class EvaluatorContext {
    private final let jexEvaluator: JexEvaluator;
    
    init(jexEvaluator: JexEvaluator) {
        self.jexEvaluator = jexEvaluator
    }
    
    public func getJexEvaluator() -> JexEvaluator {
        return jexEvaluator
    }
}
